packages/typespec-test/test/NetworkAnalytics.Management/generated/typespec-ts/samples-dev/dataProductsCreateSample.ts

```diff
@@ -0,0 +1,75 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { NetworkAnalyticsClient } from "@azure/arm-networkanalytics";
+import { DefaultAzureCredential } from "@azure/identity";
+
+/**
+ * This sample demonstrates how to create data product resource.
+ *
+ * @summary create data product resource.
+ * x-ms-original-file: 2023-11-15/DataProducts_Create_MaximumSet_Gen.json
+ */
+async function dataProductsCreateMaximumSetGen() {
+  const credential = new DefaultAzureCredential();
+  const subscriptionId = "00000000-0000-0000-0000-00000000000";
+  const client = new NetworkAnalyticsClient(credential, subscriptionId);
+  const result = await client.dataProducts.create(
+    "aoiresourceGroupName",
+    "dataproduct01",
+    {
+      properties: {
+        provisioningState: "Succeeded",
```

Comment:
is provisioningState not a readOnly property?

Reply:
yes.